ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/data/PurchaseInformation.kt

```diff
@@ -56,7 +55,7 @@ internal data class PurchaseInformation(
             subscribedProduct?.let { PriceDetails.Paid(it.price.formatted) } ?: PriceDetails.Unknown
         },
         isSubscription = transaction is TransactionDetails.Subscription && transaction.store != Store.PROMOTIONAL,
-        managementURL = managementURL,
+        managementURL = (transaction as? TransactionDetails.Subscription)?.managementURL,
```

Comment:
FFTI but should we fallback to the CustomerInfo's managementUrl? I guess that might only apply for non-subscriptions? In which case, it might be ok to just leave it as null, so FFTI.

Reply:
yes, we actually hide the "Cancel" button for non-subscriptions so it will be ignored